docs/platforms/kotlin-multiplatform/native-access-sdk.mdx

```diff
@@ -27,7 +27,10 @@ cocoapods {
   ios.deploymentTarget = "14.1"
   podfile = project.file("../iosApp/Podfile")
   // Make sure you use the proper version according to our Cocoa SDK Version Compatibility Table.
-  pod("Sentry", "~> 8.20.0")
+  pod("Sentry") {
+    version = "8.25.0"
```

Comment:
`m`: do we want to lock this to 8.25.0 or <9.0.0?

Reply:
We could do `~> 8.25` I think then we wouldn't always need to update these code snippets

Reply:
sounds reasonable to me